docs/docs/overview.md

```diff
@@ -61,30 +69,72 @@ kind: GrafanaDashboard
 metadata:
   name: grafanadashboard-sample
 spec:
-  resyncPeriod: 30s
   instanceSelector:
     matchLabels:
       dashboards: "grafana" # Notice the label
   json: ...
 ```
 
-## Cross namespace Grafana instances
+As it is a normal label selector, it is possible to leave it empty to match everything.
+
+```yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: match-all-in-namespace
+spec:
+  instanceSelector: {}
+  json: ...
+```
+
+Additionally, the more powerful `matchExpressions` are available and can be used alone or in combination with `matchLabels`
+
+```yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: match-expressions
+spec:
+  instanceSelector:
+    matchExpressions:
+    - { key: dashboards, operator: In, values: ["grafana", "loki", "tempo"]}
+    - { key: datasources, operator: NotIn, values: ["internal"]}
+  json: ...
+```
+
+## AllowCrossNamespaceImport
+
+Allows resources in one namespace to be applied to Grafana instances in other namespaces as well.
 
-To avoid exposing possible sensitive data sources to arbitrary instances, resources only affect instances in the same namespace.
+```yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: match-across-namespaces
+spec:
+  allowCrossNamespaceImport: true
+  instanceSelector: {}
+  json: ...
+```
 
-In case you need this functionality for your specific setup, use the `spec.allowCrossNamespaceImport` field of the Grafana instance.
-This setting allows resources in arbitrary namespaces to be applied to the Grafana instance.
+Disabled per default to avoid exposing possibly sensitive resources by accident.
```

Comment:
```suggestion
Disabled by default to avoid exposing possibly sensitive resources by accident.
```

Reply:
I've seen both used, but it looks like by default is the more correct way of phrasing it, I've changed it according to your suggestions :)